lib/oli/accounts.ex

```diff
@@ -252,6 +252,30 @@ defmodule Oli.Accounts do
     |> Repo.insert()
   end
 
+  def update_user(
```

Comment:
I'm wondering if we could make some changes to this function to make it more compact. One suggestion is as follows:

```elixir
def update_user(
        %User{} = user,
        %{"current_password" => _, "password" => _, "password_confirmation" => _} = attrs
      ) do
    user
    |> User.update_user_changeset(attrs)
    |> Repo.update()
    |> case do
      {:ok, %User{id: user_id}} ->
        AccountLookupCache.delete("user_#{user_id}")

      error ->
        error
    end
  end
```

This way we put in one-line the matching we want. Also the `case` is a continuation of the 2 pipes, saving one binding.

Something similar can be done for the author one.

Another comment is that now that we have 2 heads for the `update_user`, the documentation needs to be updated.